samples/cellular/nrf_cloud_multi_service/overlay-coap_nrf_provisioning.conf

```diff
@@ -0,0 +1,40 @@
+# Enable provisioning library/provisioning shell dependencies
+CONFIG_MODEM_ATTEST_TOKEN=y
+
+# Enable the provisioning library
+CONFIG_NRF_PROVISIONING=y
+CONFIG_NRF_PROVISIONING_AUTO_INIT=n
+CONFIG_NRF_PROVISIONING_COAP=y
+
+# Wait for credential installation to be complete before connecting to nRF Cloud
+CONFIG_NRF_CLOUD_CHECK_CREDENTIALS=y
+CONFIG_NRF_CLOUD_CLIENT_ID_SRC_INTERNAL_UUID=y
+
+# Configure provisioning message formatting
+CONFIG_NRF_PROVISIONING_CODEC=y
+CONFIG_NRF_PROVISIONING_CBOR=y
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+
+# Include provisioning service certificate as part of the binary.
+# Used only if none has been provisioned
+CONFIG_NRF_PROVISIONING_WITH_CERT=y
+CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG=42
```

Comment:
```suggestion
# Use a sec tag different than the HTTP overlay since CoAP uses a different CA certificate
CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG=51
```

Reply:
I used 42 because the nrf_provisioning sample uses 42 for CoAP.  I was following their lead.

Reply:
i suggested 51 since other NCS samples also use 42.

Reply:
I know your reason. I am aware of the other use of 42. I stand by my answer.